pkgs/test/nixpkgs-check-by-name/README.md

```diff
@@ -10,8 +10,13 @@ This API may be changed over time if the CI workflow making use of it is adjuste
 
 - Command line: `nixpkgs-check-by-name [--base <BASE_NIXPKGS>] <NIXPKGS>`
 - Arguments:
-  - `<BASE_NIXPKGS>`: The path to the Nixpkgs to check against
-  - `<NIXPKGS>`: The path to the Nixpkgs to check
+  - `<NIXPKGS>`: The path to the Nixpkgs to check.
+  - `<BASE_NIXPKGS>`: The path to the Nixpkgs to use as the base to compare `<NIXPKGS>` against.
+    This allows the strictness of checks to increase over time by only preventing _new_ violations from being introduced,
+    while allowing violations that already existed.
```

Comment:
At work, we call this a [ratchet linter](https://qntm.org/ratchet).

Reply:
"Ratchet" is a great term for this idea, I updated the PR to use it! https://github.com/NixOS/nixpkgs/pull/272395/commits/79618ff8cbfb69b6eb70dbc591b42cee2fed974e